cloudify_ansible/__init__.py

```diff
@@ -94,6 +100,11 @@ def wrapper(playbook_path=None,
         _get_instance(ctx).update()
 
         try:
+            extra_packages = extra_packages or _get_node(ctx).properties.get(
+                'extra_packages') or []
+            extra_packages.append(ANSIBLE_TO_INSTALL)
```

Comment:
I don't see why this is necessary? To make execution time longer?

Reply:
We must install ansible in the venv.
if using type executor then the input directly to function.
if using the playbook node type then this information is  in properties.